### DISCUSSION TITLE
Ability to customize the strings displayed (currently "Default Media Receiver" and "Desktop Audio Streamer" on my device)

### DISCUSSION THREAD

**Discussion by mikecarnohan** [SamDel/ChromeCast-Desktop-Audio-Streamer#157] [Ideas]
It would be nice to have an option in settings to (re)name the strings for "Default Media Receiver" (if possible) and "Desktop Audio Streamer", so that users could name their streaming server something of their own choosing. In situations where audio is being cast to device like a SmartTV, these strings leave something to be desired as they are not configurable.

**Comment by SamDel** [comment]
It's possible to create an option for "Desktop Audio Streamer". It's defined [here in code](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/blob/bb6dcd760e2aeb43bce0e7104e34c1a180f5165d/Source/ChromeCast.Desktop.AudioStreamer/Properties/Strings.resx#L166).

"Default Media Receiver" is defined by Chromecast.

**Comment by SamDel** [comment]
I added an option in [ChromecastDesktopAudioStreamer 5.3.2.zip](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/files/14557103/ChromecastDesktopAudioStreamer.5.3.2.zip) to change the title of the stream. This title will show up on your SmartTV etc.

The title is only send to the device when the stream is loaded, so when you change the title it will only show up after you (re)started the stream!

Can you try?

**Comment by SamDel** [comment]
This is implemented in the [5.4 release](https://github.com/SamDel/ChromeCast-Desktop-Audio-Streamer/releases/tag/v5.4).
